.github/workflows/bump-dependencies.yml

```diff
@@ -0,0 +1,174 @@
+name: Bump dependencies
+
+on:
+  # For PR creation:
+  workflow_dispatch:
+  schedule:
+    - cron: '13 8 * * 6'  # once a week on Saturdays at 08:13
```

Comment:
Any reason for that time? Usually maintenance takes place at night, I think? (It doesn't matter much though...).

Reply:
It's rather arbitrary. I'd avoid the standard minutes 0/15/30/45 as those are usually loaded (at least in my experience from managing larger fleets of servers). Haven't thought about the exact hour a lot, but my thinking was that main usage is probably dominated by other timezones anyway.